src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -130,15 +119,20 @@ public class BlobFileStore : IFileStore
         EnsureArg.IsNotNullOrWhiteSpace(blockId, nameof(blockId));
 
         BlockBlobClient blobClient = GetInstanceBlockBlobClient(version);
+        _logger.LogInformation("Trying to read block list for DICOM instance file with watermark '{Version}'.", version);
```

Comment:
Should we set the log level on these statements to `Debug`? This will be a lot of logs.

Reply:
Good point. There is no limit right on logs right now. Lets dial it down when we have to.